### PR TITLE
added a property check in renderableModelEntity for the animation url…

### PR DESCRIPTION
**Pull request by amantley**
… this fixes the bug stopping you from changing animations on an entity

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6405/app_12202_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6405/HighFidelity-Beta-PR12202-4eb2a5cf5e900a80abda46677038b0e88de53445.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6405/HighFidelity-Beta-PR12202-4eb2a5cf5e900a80abda46677038b0e88de53445.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6410/app_12202_PR.apk

**Comment by amantley** [comment]
Test Plan 2957 is at https://highfidelity.testrail.net/index.php?/cases/view/2957

Step 11 (listed here) was added to cover this bug:

Preconditions:
Download the attached filed to your local system: Mason_Idle.fbx, right12.fbx, and Mason_Idle.fst
Add Mason_Idle.fbx to your asset server
Add right12.fbx to your asset server
Add Mason_Viewing mirror.fbx to the asset server
Create menu-> open this domain's asset server -> choose file
Then under the properties for Mason_Idle.fbx set the animation url to be atp:/right12.fbx file.

Step 11:  In the animation url box, type in the url for the animation Mason_Viewing mirror.fbx
"atp:/Mason_Viewing mirror.fbx", hit return to update the property.

Expected Result:  The animation should change from sitting down to standing and looking in a mirror. All the above steps should work for the newly loaded animation.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6429/app_12202_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6429/HighFidelity-Beta-PR12202-1c97272767e3806c3fa9aa35d9474097c57e9934.dmg

File: libraries/entities-renderer/src/RenderableModelEntityItem.cpp [hunk 1]
```diff
@@ -1394,9 +1394,16 @@ void ModelEntityRenderer::doRenderUpdateSynchronousTyped(const ScenePointer& sce
     // That is where _currentFrame and _lastAnimated were updated.
     if (_animating) {
         DETAILED_PROFILE_RANGE(simulation_physics, "Animate");
+        
         if (!jointsMapped()) {
             mapJoints(entity, model->getJointNames());
         }
+        //else the joints have been mapped before but we have new animation to load
+        else if (_animation && (_animation->getURL().toString() != entity->getAnimationURL())) {
```
Comment (ZappoMan) [inline libraries/entities-renderer/src/RenderableModelEntityItem.cpp]: coding standard: brace placement `} else if (...) {`

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6434/app_12202_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6434/HighFidelity-Beta-PR12202-961ed88fb094515e95afd656ba2e920073cabfa5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6434/HighFidelity-Beta-PR12202-961ed88fb094515e95afd656ba2e920073cabfa5.dmg

**Comment by QAadahlin** [comment]
Currently Testing

**Comment by QAadahlin** [comment]
Completed Testing. PASSED

**Comment by QAadahlin** [comment]
@conklin94122 Completed testing. PASSED

**Pull request merged**